.github/workflows/ci-go.yaml

```diff
@@ -40,3 +42,8 @@ jobs:
         with:
           dir: file://atlasaction/testdata/migrations
           url: sqlite://file.db?mode=memory
+      - uses: ./migrate/push
```

Comment:
Make an assertion on the output of the action (I need to add it to the `apply` action as well). Suited for a follow up small PR. 

Example: 
https://github.com/ariga/atlas-deploy-action/blob/d15abdf18a253c896d6afd707b671182a1b8ccbe/.github/workflows/ci-go.yaml#L71

Reply:
Working on it. The rest of the PR is ready for re-review

Reply:
Done.